src/plugins/pbjx/utils/filterRemoved.js

```diff
@@ -0,0 +1,27 @@
+import isPlainObject from 'lodash/isPlainObject';
+
+/**
+ * Removes "[removed]" entries from nodes changeset.
+ */
+const filterRemoved = (data) => {
+  if (!isPlainObject(data)) {
+    return data;
+  }
+  return Object.keys(data).reduce((accumulator, index) => {
```

Comment:
@rsumilang nit: i'd probably change `index` here to `key` since that's what it is